main.go

```diff
@@ -224,4 +226,10 @@ func main() {
 	} else {
 		log.Fatal(http.ListenAndServe(*listenAddress, exp))
 	}
+
+	s := make(chan os.Signal, 1)
+	signal.Notify(s, syscall.SIGINT, syscall.SIGTERM)
+	_sig := <-s
```

Comment:
Thanks for the PR!

How does this fix the issue? 
The listen call in line 225/227 should block so I don't think we ever get here.